selfapi.js

```diff
@@ -6,6 +6,13 @@ var https = require('https');
 var nodepath = require('path');
 var url = require('url');
 
+function ensureSerialized(strOrObject, spaces=null) {
+  if (strOrObject instanceof Object) {
+    return JSON.stringify(strOrObject, null, spaces);
+  }
+  return strOrObject;
```

Comment:
It's probably a very small edge case anyway (I don't know any API handlers out there that need to return an un-wrapped JSON string including the quotes), and I think there is more value in your `ensureSerialized` mechanism than in addressing this unlikely edge case. Let's ship your feature! 😄 Thanks again for doing this.